src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -106,16 +143,42 @@
   </ItemGroup>
   <!-- WINDOWS: CoreCLR -->
   <ItemGroup Condition=" '$(TargetsWindows)' == 'true' And '$(CoreClrOrCorRt)' == 'true'">
+    <Compile Include="System\Environment.CoreCLR.cs" />
```

Comment:
I guess I'm confused when we're choosing to use .NETNative.cs/.CoreCLR.cs vs .Win32.cs/.WinRT.cs (we have a bunch of files in the corefx repo that use the former convention, and I was just following that).  But I'll switch them.